pyannote/database/custom.py

```diff
@@ -271,6 +293,87 @@ def subset_iter(
             {"uri": uri, "database": database, "subset": subset}, lazy=lazy_loader
         )
 
+def subset_trial(
+    self,
+    database: Text = None,
+    task: Text = None,
+    protocol: Text = None,
+    subset: Text = None,
```

Comment:
Type should be `Subset` (defined in `pyannote.database`)

Reply:
Can't find where it is defined to import it :thinking:

Reply:
In `pyannote.database.protocol.protocol`

Reply:
So I should do it for those 3 :
```
def meta_subset_iter(
...
    meta_subset: Text, # change to Subset
    subset_entries: Dict,
    database_yml: Path,
):
...
def subset_iter(
...
    subset: Text = None, # change to Subset
    entries: Dict = None,
    database_yml: Path = None,
):
...
def subset_trial(
...
    subset: Text = None, # change to Subset
    entries: Dict = None,
    database_yml: Path = None,
):
```
?

Reply:
Yes, thanks.